src/components/shared/SEOMetaTag/index.tsx

```diff
@@ -14,22 +14,28 @@ const MetaTag = (props: Props) => {
 			<title>{props.title ? props.title : '알티모빌리티 - 차량대여 서비스'}</title>
 			<meta
 				name="description"
-				content={props.description ? props.description : '알티모빌리티 - 차량대여 서비스'}
+				content={props.description ? props.description : '세상과 사람, 자동차의 교집합'}
 				data-react-helmet="true"
 			/>
-			<meta name="keywords" content="자동차" />
+			<meta name="keywords" content="자동차, Freemium, Sharing, Big Data, Economy" />
 			<meta property="og:type" content="website" />
 			<meta property="og:title" content={props.title ? props.title : '알티모빌리티 - 차량대여 서비스'} />
 			<meta property="og:site_name" content="알티모빌리티 - 차량대여 서비스" />
 			<meta property="og:description" content={props.description ? props.description : '알티모빌리티 - 차량대여 서비스'} />
```

Comment:
```suggestion
			<meta property="og:description" content={props.description ? props.description : '세상과 사람, 자동차의 교집합'} />
```

og graph default description 도 변경하면 좋겠네요 ~